Data/Sys/GameSettings/GJX.ini

```diff
@@ -0,0 +1,16 @@
+# GJXE51, GJXP51 - Vexx
+
+[Core]
+# Values set here will override the main Dolphin settings.
+
+[OnFrame]
+# Add memory patches to be applied every frame here.
+
+[ActionReplay]
+# Add action replay cheats here.
+$Disable Culling
+04011A14 60000001
+04011A18 4E800020
```

Comment:
Are you sure this code works on all versions of the game? If not, it should be in a six-character INI file.

Reply:
The wiki does not specify. https://wiki.dolphin-emu.org/index.php?title=Vexx I am unable to verify this because I don't own a copy of it.

Reply:
I'm not comfortable with this unless someone verifies it works in both.